taccsite_cms/static/site_cms/css/src/_imports/elements/html-elements.css

```diff
@@ -15,6 +15,12 @@ Reference:
 Styleguide Elements.MainSectioningRoots
 */
 
+html {
+  /* Set base font but support user-defined browser font size */
+  /* SEE: https://snook.ca/archives/html_and_css/font-size-with-rem */
+  font-size: 100%; /* 16px */
+}
```

Comment:
**Noop Placeholder Code**

I normally set this to 62.5% to achieve `1px` = `1rem`, but I'm waiting untill Frontera Redesign's issue #98.